dapps/marketplace/src/components/DesktopUserActivation.js

```diff
@@ -443,6 +473,11 @@ class UserActivation extends Component {
     )
   }
 
+  refetchQueries() {
+    this.props.identityRefetch()
+    this.props.client.reFetchObservableQueries()
```

Comment:
do we need both of these? Does reFetchObservableQueries also cause an identity refetch?

Reply:
`reFetchObservableQueries()` is enough. It does refetch the identity. Will remove `this.props.identityRefetch()`